tutorials/quantum_simulation/Shadow estimation for VQS.ipynb

```diff
@@ -1,20 +1,5 @@
 {
```

Comment:
<p>The <code>\ket</code>don't render on the preview. To be checked on the final render.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>